cluster/gce/util.sh

```diff
@@ -2464,7 +2485,18 @@ function kube-up() {
     fi
     check-cluster
 
+    ## TODO: RP, or TP kubeconfig naming convention
+    ##       if the convention and naming is agreed upon all components, the COPY can be removed and the start-kubemark
+    ##       script can use the kubeconfig created here
+    ##
     if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
+      if [ -z "${LOCAL_KUBECONFIG:-}" ]; then
+        echo "Local_kubeconfig not set"
+      else
+        cp -f ${KUBECONFIG} ${LOCAL_KUBECONFIG}
```

Comment:
why do we need this? i cleaned up a lot of this i think

Reply:
outdated comments, this has been fixed i believe.